lib/fixture_tree.rb

```diff
@@ -99,7 +99,7 @@ def merge(data)
       end
     else
       delete
-      @path.write(data)
+      @path.open("w") {|f| f << data }
```

Comment:
Use `wb` to avoid CRLF <-> LF issues on Windows, and add a space after the `{` for stylistic consistency. Other than that, lgtm.

Reply:
done, sorry for the slow response